frontend/src/app/forms/steps/pictures-entry/pictures-entry.component.html

```diff
@@ -0,0 +1,45 @@
+<div class="main-description">
+  <h1>Picture</h1>
+
+  <!--<ion-img [src]="photo"></ion-img>-->
+  <div
+    *ngIf="
+      formsData.controls.pictureEntry.controls.picString.invalid;
+      else elseBlock
+    "
+  >
+    <span>
+      Add a picture of yourself by tapping on the image icon.
+    </span>
+    <ion-button
+      (click)="takePicture()"
+      class="custom-button"
+      fill="clear"
+      size="large"
+    >
+      <ion-icon name="person-circle"></ion-icon>
+    </ion-button>
+  </div>
+  <ng-template #elseBlock>
+    <span>
+      Good job! If you want to take another picture, just delete the actual
+      picture by sliding on it and take another one.
+    </span>
+    <ion-item-sliding class="ion-no-padding custom-sliding">
+      <ion-item-options side="end">
+        <ion-item-option
+          color="danger"
+          expandable
+          (click)="deletePictureEntry()"
+        >
+          <ion-icon name="trash" size="large"></ion-icon>
+        </ion-item-option>
+      </ion-item-options>
+      <ion-item class="ion-no-padding">
+        <ion-img
+          [src]="formsData.controls.pictureEntry.controls.picString.value"
+        ></ion-img>
```

Comment:
Is there any advantage when using ion-img instead of a simple img?

Reply:
I just used it because its from ionic. So should we better use just img? it works the same in this case @MBuchalik

Reply:
I think we can just use a normal img here. If I understood it correctly in their docs, ion-img is rather a thing that can be used for large lists of images.

(But honestly, there shouldn't be a huge difference in our case. So if you want to keep ion-img, then that's fine as well.)